sigh/spec/runner_spec.rb

```diff
@@ -151,6 +151,23 @@
         expect(devices.size).to eq(1)
       end
 
+      it "devices for development with apple silicon" do
+        options = { development: true, include_mac_in_profiles: true }
+        Sigh.config = FastlaneCore::Configuration.create(Sigh::Options.available_options, options)
+
+        ios_device = "ios_device"
+        allow(ios_device).to receive(:id).and_return(1)
+        allow(ios_device).to receive(:device_class).and_return(Spaceship::ConnectAPI::Device::DeviceClass::IPHONE)
+        as_device = "as_device"
+        allow(as_device).to receive(:id).and_return(2)
+        allow(as_device).to receive(:device_class).and_return(Spaceship::ConnectAPI::Device::DeviceClass::APPLE_SILICON_MAC)
+
+        expect(Spaceship::ConnectAPI::Device).to receive(:all).and_return([ios_device, as_device])
+
+        devices = fake_runner.devices_to_use
+        expect(devices.size).to eq(2)
+      end
```

Comment:
This works but I kinda have the feeling you're `allow`ing too many statements here 😬 not sure if this really attests the functionality you implemented, what do you think?

Reply:
Ping @PinkidG , WDYT? :)

Reply:
Hey @rogerluan Thanks for the feedback, I'm sorry I didn't get back to it. I'm not really a ruby expert tbh. 😅

I'll try to improve it 🙂